Tribute_Page/Readme.md

```diff
@@ -0,0 +1,46 @@
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [My-Portfolio] 
+
+**[My-Portfolio]** is a simple Trial page for code riviewing proccess
+
+
+## 🛠 Built With <a name="built-with"></a>
+  html, css 
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
```

Comment:
while you use a professional readme, it is not clearly presented please try to make it clear and leave out unnecessary parts.

Reply:
> while you use a professional readme, it is not clearly presented please try to make it clear and leave out unnecessary parts.

- [ ] It is always important to add the WHY, this is for the student to know the reason you are requesting changes.